### PR TITLE
Make pyjsonviewer  callable from python.

### LINKED ISSUE #33: Make pyjsonviewer callable from python code.
Expectation: Ability to visualise json files as well as directly load json objects from within python code.

### PR DESCRIPTION
**Pull request by Ankush-Chander**
Hi @AtsushiSakai,
 
This PR fixes #33 

- [x] refactor main file
- [x] modify setup.py
- [x] update readme file

Please review.

### CODE REVIEW

**Comment by AtsushiSakai** [comment]
Could you rebase this PR based on latest branch?

**Comment by Ankush-Chander** [comment]
Please check.

**Comment by Ankush-Chander** [comment]
Ughh.
I think I confused upstream/origin master and it has bloated up number of commits in the PR.

File: README.md [hunk 1]
```diff
@@ -52,6 +52,26 @@ or download as zip.
...
 # How to use
 
+## Inside python code
+
+### View json file
+```
+import pyjsonviewer
+
+pyjsonviewer.view_data(json_file="dat/list.json")
+
```
Comment (AtsushiSakai) [inline README.md]: Just a question. What is a possible use case? Why do you want to call this module from your code?

File: README.md [hunk 2]
```diff
@@ -52,6 +52,26 @@ or download as zip.
 
 # How to use
 
+## Inside python code
```
Comment (AtsushiSakai) [inline README.md]: Could you pleas move this section after "Menu bar function"?

**Review by AtsushiSakai — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Ankush-Chander) [reply README.md]: I am an IR(Information Retrieval) researcher. It requires me to frequently compare older results with new results and view them conveniently.

Currently I had to log results and copy them in online json-viewers or save them in a file and view it in the editor.

Being able to invoke `pyjsonviewer` from code helps me quickly check results without the above hassle.

my [long term goal](https://github.com/Ankush-Chander/irlab) is to have a fullfldeged Desktop app that makes this comparing results, producing reports, disseminating the analysis report across team super easy for IR researchers.

**Review by AtsushiSakai — APPROVED** [review]
LGTM.

**Comment by Ankush-Chander** [comment]
Thank you.

**Pull request merged**